etc/p3-config/p3-config.yml

```diff
@@ -329,34 +329,44 @@ p3_flavor_compute_A:
   ram: 131072
   disk: 371
   vcpus: 64
+  extra_specs:
+    "resources:CUSTOM_COMPUTE_A": 1
```

Comment:
Shouldn't this be CUSTOM_CUSTOM_COMPUTE_A? :p

Reply:
No :p